examples/TwoHundredFields.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Formik, FastField, Form } from 'formik';
+import { unstable_trace as trace } from 'scheduler/tracing';
+
+const sleep = ms => new Promise(resolve => setTimeout(resolve, ms));
+
+const onSubmit = async values => {
+  await sleep(300);
```

Comment:
Never seen this as a good idea to deliberately slow down tests, even if it's for a bunch of ms. Jest timer mocks are probably a better option. Besides, it feels like some arbitrary number that can brake tests.

Reply:
It's not a test. It's an example.